parlai/tasks/wrapper/agents.py

```diff
@@ -23,37 +23,19 @@
 from parlai.core.teachers import create_task_agent_from_taskname, Teacher
 
 
-class LabelToTextTeacher(Teacher):
+class AbstractWrapperTeacher(Teacher):
```

Comment:
maybe make this an abstract base class (ABC) since it has abstract methods that need to be implemented by children https://docs.python.org/3/library/abc.html